src/compile/build.ts

```diff
@@ -236,6 +236,10 @@ function spawnProcess(step: Step): ProcessEnv {
         if (step.command === 'latexmk' && step.rootFile === lw.root.subfiles.path && lw.root.dir.path) {
             cwd = lw.root.dir.path
         }
+        if(step.command==='bibtex')
+        {
+            env['openout_any'] = 'a'
+        }
```

Comment:
Even though I understand the point of adding this, it cannot be done without warning the users about the potential security breach. I am against merging such a change, but it could documented in the wiki or even in the _Discussions_ tab.